script/SharedGovernorConstants.sol

```diff
@@ -26,9 +27,8 @@ contract SharedGovernorConstants {
     uint256 public constant L1_TIMELOCK_MIN_DELAY = 259_200;
     address public constant L1_ARB_ONE_DELAYED_INBOX = 0x4Dbd4fc535Ac27206064B68FfCf827b0A60BAB3f;
 
-    address public constant L2_CORE_GOVERNOR_NEW_DEPLOY = 0x7796F378B3c56ceD57350B938561D8c52256456b;
-    address public constant L2_TREASURY_GOVERNOR_NEW_DEPLOY =
-        0x4fd1216c8b5E72b22785169Ae5C1e8f3b30C19E4;
+    address public constant L2_CORE_GOVERNOR_NEW_DEPLOY = address(0);
```

Comment:
why set these to 0? and why move away from this pattern?

Reply:
Because we don't yet have a deployed version of these contracts.